main.tf

```diff
@@ -1,15 +1,26 @@
 data "aws_region" "current" {}
 
-provider "aws" {
-  default_tags {
-    tags = merge(
-      {
-        Release_Name = var.resource_name_prefix
-        Name         = "${var.resource_name_prefix}-graphdb"
-      },
-      var.common_tags
-    )
-  }
+# Fetch availability zones for the current region
+data "aws_availability_zones" "available" {}
+
+locals {
+  azs = slice(data.aws_availability_zones.available.names, 0, 3)
```

Comment:
Not needed as this is not used in the vpc module, AFAICS

Reply:
yes, this seems duplicated inside the vpc module. Either remove it from here and get them inside the vpc module, or if any other module needs them - get them here and pass them along.